build.sbt

```diff
@@ -4,6 +4,16 @@ organization in ThisBuild := "me.shadaj"
 
 scalaVersion in ThisBuild := "2.12.8"
 
+Compile / scalaSource := baseDirectory.value / "src" /  "main" / "scala"
+
+Test / scalaSource := baseDirectory.value / "src" / "test" / "scala"
+
+// dependencies for scalapy-tensorflow-source
+libraryDependencies += "me.shadaj" %%% "scalapy-core" % "0.3.0"
```

Comment:
Actually... how come could this project ever compile before without these deps? :hushed: or maybe: why are they only required by IntelliJ (at least that's how I understand the PR title), but not by a standalone sbt?

Reply:
Ok sooo this project is complicated.
If you look at the project structure you will see scalapy-tensorflow-soruces without dependecies and then JVM and Native versions with dependencies. Look below on `scalaPyTensorFlowCross`.
So we can say that all deps are production deps and there are no dev deps so intellij is blind.

Reply:
This shouldn't be necessary, the dependency is brought in transitively by the dependency on `scalapy-numpy`.